adapters/adhese/adhese.go

```diff
@@ -0,0 +1,247 @@
+package adhese
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AdheseAdapter struct {
+	http             *adapters.HTTPAdapter
```

Comment:
This is an outdated interface only used for the legacy endpoint. You probably saw this in an older adapter. Since it doesn't look like you're using it anyway, please remove it.